src/Microsoft.ML.Transforms/GroupTransform.cs

```diff
@@ -180,35 +180,42 @@ public override RowCursor[] GetRowCursorSet(Func<int, bool> predicate, int n, Ra
         }
 
         /// <summary>
+        /// The output schema collects columns used to group columns (aka group-key columns) columns being grouped from the source scheam.
+        /// In an output schema, group-key columns are followed by grouped columns. For example, if column "Age" is used to group "UserId" column,
```

Comment:
>grouped [](start = 67, length = 7)

I would call them aggregated columns.

Reply:
Sure.

---
In reply to: [245808905](https://github.com/dotnet/machinelearning/pull/2060#discussion_r245808905) [](ancestors = 245808905)